src/app/Core/api.service.ts

```diff
@@ -0,0 +1,20 @@
+export interface ApiResponse<T>{

```

Comment:
this should also be typed: `interface ApiResponse<T extends Object>`

Reply:
Done.